rxv/rxv.py

```diff
@@ -124,6 +125,30 @@ def on(self, state):
     def off(self):
         return self.on(False)
 
+    def play(self):
+        self._send_action('Play')
+
+    def pause(self):
+        self._send_action('Pause')
+
+    def stop(self):
+        self._send_action('Stop')
+
+    def next(self):
+        self._send_action('Skip Fwd')
+
+    def previous(self):
+        self._send_action('Skip Rev')
+
+    def _send_action(self, action):
+        src_name = self._src_name(self.input)
+        if not src_name:
+            return None
```

Comment:
I'm not sure, but I'm feeling we should probably raise some kind of exception here `PlaybackUnavailable` or sth when sth goes wrong.

Also having a function to check if playback actions are available would also make API simpler to use.